requirements_bundles.txt

```diff
@@ -0,0 +1,9 @@
+# These are the requirements that the extension bundle
+# loading mechanism need on Python 2 and can be removed
+# when moved to Python 3.
+# It's automatically installed when running npm run bundle
+
+# These can be removed when upgrading to Python 3.x
+importlib-metadata==0.9  # remove when on 3.8
+importlib_resources==1.0.2  # remove when on 3.7
+pathlib2==2.3.3  # remove when on 3.x
```

Comment:
Why the separate requirements file? Considering that these requirements are mandatory for running Redash, feels like this will only create confusion/issues.

Reply:
Oh I missed this comment. In the CI steps that run frontend related tests we want to run the bundle script before, so we'll need to install the requirements. I thought it would be less expensive to simply only those that are needed for the bundle script.

Reply:
🤔 how about we add to `requirements.txt` a directive to install this one (`-r requirements_bundles.txt`), to minimize the affect on the rest of the configuration?

Reply:
Sure, that works for me!

Reply:
Done in https://github.com/getredash/redash/pull/3569/commits/eb9eb3be48c97d8b8184dcfd0917b595a55dbd93